examples/crd-client/Dockerfile

```diff
@@ -14,21 +14,18 @@
 
 # Gather dependencies and build the executable
 FROM golang:1.20.4 as builder
-WORKDIR /go/src/crd-client
+WORKDIR /
```

Comment:
I'm curious why you changed the paths in the build container. That shouldn't affect the location in the final container.

Reply:
Mark suggested moving the `WORKDIR` path to the root directory ('/'). 
Example PR: https://github.com/googleforgames/agones/pull/3270/files

Reply:
I was being consistent with everything else I've seen with distroless / done with distroless.

There's no $HOME folders in distroless, so it seems like everyone drops everything in /.

Reply:
Oh sorry, I should have looked at this properly - this is the build step.

🤷🏻 actually for this, I have no strong opinion.

Reply:
Based on testing, the change to the root directory ('/') for WORKDIR in the build container didn't work as expected, so I've reverted it back.